src/Joomlatools/Console/Command/Site/Configure.php

```diff
@@ -135,20 +135,21 @@ protected function _configureJoomlaCMS()
 
         $contents = file_get_contents($source);
         $replace  = function($name, $value, &$contents) {
-            $pattern = sprintf('#\$%s\s*=\s*(["\']).*?(?<!\\\\)\1#', $name);
+            $pattern = sprintf('#\$%s\s*=\s*(["\']?).*?\1(?=[;\1])#', $name);
             $match   = preg_match($pattern, $contents);
+            $value   = ctype_digit($value) ? $value : "'" . str_replace("'", "\\'", $value) . "'";
```

Comment:
@msaladna Instead of `str_replace()` we can use `addslashes()` here, and have all quote styles covered?

Reply:
@msaladna Oops, please ignore my previous comment. `addslashes()` is not a good idea as it would put the `\` character in there literally. `str_replace()` is what we need indeed!